build/Dockerfile

```diff
@@ -1,26 +1,27 @@
 FROM quay.io/app-sre/boilerplate:image-v2.3.2 AS builder
-
-RUN mkdir -p /workdir
-WORKDIR /workdir
-COPY go.mod go.sum ./
+FROM golang:1.17 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
 RUN go mod download
-COPY . .
```

Comment:
The removal of this means it won't be pulling in directories such as `pkg`, `config`, `version` and `util` which is what will be causing build problems in the CI space.

Currently it looks like it's only copying `api` and `controllers`.

Should maybe just revert to a copy of `.`.

Reply:
Thanks Matt! Have updated the Dockerfile with the same content as pre-migrated version. That should work mostly.